cmd/main.go

```diff
@@ -26,9 +27,11 @@ import (
 )
 
 var cmd = &cobra.Command{
-	Use:   "kapply",
-	Short: "Perform cluster operations using declarative configuration",
-	Long:  "Perform cluster operations using declarative configuration",
+	Use:           "kapply",
+	Short:         "Perform cluster operations using declarative configuration",
+	Long:          "Perform cluster operations using declarative configuration",
+	SilenceErrors: true,
+	SilenceUsage:  true,
```

Comment:
Just to be clear... We are doing this here because kapply is not part of kpt ? All the other commands are not doing this because they will be silenced in kpt ?

Reply:
We want to handle error reporting ourselves instead of relying on Cobra. That allows us to map errors to error message and handle known errors in a single place.